services/ui-src/src/components/drawers/ReportDrawerDetails.tsx

```diff
@@ -42,7 +42,45 @@ export const ReportDrawerDetails = ({ entityType, drawerDetails }: Props) => {
         </Box>
       );
     case ModalDrawerEntityTypes.QUALITY_MEASURES:
-      return <Text sx={sx.detailDescription}>Quality Measures TODO</Text>;
+      return (
+        <Box sx={sx.detailBox}>
+          <Text sx={sx.detailSubtitle}>Measure Domain</Text>
+          <Text sx={sx.detailSubtext}>{drawerDetails.domain}</Text>
+          <Grid gap="normal 6" templateColumns="33% 66%">
```

Comment:
Nit: We don't use grid elsewhere for this type of thing so I'm tempted to say we should keep it the same for the sake of consistency. I'm not necessarily opposed to it, though.